### PR TITLE
Describe more signals in the schema

### PR DESCRIPTION
**Pull request by ilfa**

### CODE REVIEW

File: schemas/fingerprint-server-api.yaml [hunk 1]
```diff
@@ -558,8 +658,115 @@ components:
...
+              $ref: '#/components/schemas/ProductError'
+        proxy:
+          title: SignalResponseProxy
+          type: object
+          properties:
+            data:
+              type: object
+              properties:
+                result:
+                  type: boolean
```
Comment (JuroUhlar) [inline schemas/fingerprint-server-api.yaml:758]: Also, can we add a description here?

File: schemas/fingerprint-server-api.yaml [hunk 2]
```diff
@@ -633,13 +846,89 @@ components:
             - notDetected
             - good
             - bad
-      example:
-        result: 'notDetected'
+          example: 'bot'
```
Comment (JuroUhlar) [inline schemas/fingerprint-server-api.yaml:849]: Shouldn't example of `result` be one of the enum values?

File: schemas/fingerprint-server-api.yaml [hunk 3]
```diff
@@ -558,8 +660,117 @@ components:
...
+              type: object
+              properties:
+                result:
+                  type: boolean
+                  description: >
+                    Android specific emulator detection. There are 2 values:
+                    • `true` - Emulated environment detected (e.g. launch inside of AVD)
+                    • `false` - No signs of emulated environment detected
+
+                    Available only for events from Android client. The field will be empty for a browser/iOS event.
```
Comment (JuroUhlar) [inline schemas/fingerprint-server-api.yaml:718]: "empty" meaning `undefined` or `null` ?

Comment (ilfa) [reply schemas/fingerprint-server-api.yaml:718]: Good question, I just copied it from the documentation

File: examples/webhook.json [hunk 4]
```diff
@@ -56,5 +56,85 @@
   "lastSeenAt": {
     "global": "2022-03-16T11:28:34.023Z",
     "subscription": null
+  },
+  "bot": {
```
[the remark this answers is not in the document]
Comment (Romchela) [reply examples/webhook.json:60]: Example is [here](https://github.com/fingerprintjs/koala/blob/master/poster/worker/test/inputs/postingformat/full_smart_signals.json#L9). See the information inside `data` field.

File: schemas/fingerprint-server-api.yaml (hunk 3 above)
Comment (JuroUhlar) [reply schemas/fingerprint-server-api.yaml:687]: I tested it, it won't be present in the response at all. So let's say that: "The field will not be present for a browser or iOS event."

Comment (ilfa) [reply schemas/fingerprint-server-api.yaml]: Sure, fixed!

**Pull request merged**